Documentation/building/advanced-inner-loop-testing.md

```diff
@@ -42,7 +42,7 @@ public class Program
 
 3. Copy the just built corefx assemblies into your application directory. When using Visual Studio or the .NET Core SDK Host (`dotnet.exe`) you usually compile against *reference assemblies*. For simplicity we compile against the same assembly set that we use during run time.
 ```
-xcopy ..\corefx\artifacts\bin\testhost\netcoreapp-Windows_NT-Release-x64\shared\Microsoft.NETCore.App\9.9.9 runtime /e /i /y /s
+xcopy ..\corefx\artifacts\bin\testhost\netcoreapp-Windows_NT-Release-x64\shared\Microsoft.NETCore.App\3.0.0 runtime /e /i /y /s
```

Comment:
Now that this changes with every release, would it make sense to have these docs reference some variable?